README.md

```diff
@@ -735,7 +735,19 @@ O ExampleProviderTest mostrado precisa fornecer o estado de acordo com o arquivo
 
 ### <a id="sec-api-end-to-end-tests"></a>REST API End-to-End Test
 
-## <a id="sec-exploratory-tests"></a>Exploratory Testing
+## <a id="sec-exploratory-tests"></a>Testes Exploratórios
+
+Mesmo os esforços de automação de teste mais diligentes não são perfeitos. Às vezes, você perde certos casos extremos em seus testes automatizados. Às vezes é quase impossível detectar um bug específico escrevendo um teste unitário. Certos problemas de qualidade nem se tornam aparentes em seus testes automatizados (pense em *design* ou usabilidade). Apesar de suas melhores intenções com relação à automação de teste, alguns tipos de testes manuais ainda são uma boa ideia.
+
+![Testes exploratórios](/assets/exploratoryTesting.png "Testes exploratórios")
+
+**Figura 12**. Use testes exploratórios para identificar todos os problemas de qualidade que seu *pipeline* de construção não detectou
+
+Inclua [testes exploratórios](https://en.wikipedia.org/wiki/Exploratory_testing) em seu portfólio de testes. É uma abordagem de teste manual que enfatiza a liberdade e a criatividade do testador para detectar problemas de qualidade em um sistema em execução. Simplesmente dedique algum tempo em uma programação regular, arregace as mangas e tente interromper sua inscrição. Use uma mentalidade destrutiva e encontre maneiras de provocar problemas e erros em seu aplicativo. Documente tudo o que encontrar para mais tarde. Fique atento a *bugs*, problemas de *design*, tempos de resposta lentos, mensagens de erro ausentes ou enganosas e tudo mais que possa incomodá-lo como usuário de seu *software*.
```

Comment:
Arregace as mangas e tente "quebrar" sua aplicação. Acho que fica melhor. Tem alguns lugares que a palavra "bug" não está em itálico, como aqui. Acho que "software" pode ficar sem itálico mesmo.